### PR TITLE
feat: Update README and add turntable job bundle sample

### PR DESCRIPTION
**Pull request by mwiebe**
### Description

* We have published a developer guide at https://docs.aws.amazon.com/deadline-cloud/latest/developerguide, and some of the content is now duplicated between here and the guide. Remove the duplicated content and replace it with a summary and http links.
* Add a summary of the various job bundle samples to the job_bundles/README.md file.
* Add a new job bundle sample, Turntable with Maya/Arnold. This job bundle takes an OBJ file as input, and produces a video turntable render as output.

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: job_bundles/turntable_with_maya_arnold/template.yaml [hunk 1]
```diff
@@ -0,0 +1,449 @@
...
+name: Turntable with Maya/Arnold
+description: |
+  This job takes an OBJ geometry file as input, and outputs a video turntable render.
+  A turntable render is a 360 degree rotation of a rendered 3D model that allows inspection
+  of the geometry without a 3D viewer. You can download just the EncodeVideo step from
+  Deadline Cloud Monitor to get the video without the intermediate scene file or images.
+
+  It uses two conda channels to get software. Maya and Arnold is acquired from the
+  deadline-cloud channel accessible from Deadline Cloud service-managed fleets, and FFmpeg
+  if from the conda-forge community channel. To run the job without such a conda virtual
```
Comment (ddneilson) [inline job_bundles/turntable_with_maya_arnold/template.yaml]:
```suggestion
  is from the conda-forge community channel. To run the job without such a conda virtual
```

File: job_bundles/README.md [hunk 2]
```diff
@@ -1,327 +1,88 @@
...
-when it runs. Deadline Cloud uses this as an interface for its
-[Job Attachments feature](https://docs.aws.amazon.com/deadline-cloud/latest/userguide/storage-job-attachments.html).
-If you do not have all of your inputs and outputs for Job Attachments listed in this file
-then you can still select them for your Job during submission with the `deadline bundle gui-submit` CLI command.
+The [gui_control_showcase](gui_control_showcase/template.yaml) job bundle shows every GUI control that user interface metadata
+on [Open Job Description job parameters](https://github.com/OpenJobDescription/openjd-specifications/wiki/2023-09-Template-Schemas#2-jobparameterdefinition)
+support.
 
-The format of the file and the purpose of each element is as follows:
+## Job development progression
```
Comment (ddneilson) [inline job_bundles/README.md]:
```suggestion
### Job development progression
```

**Review by crowecawcaw — DISMISSED** [review]

**Review by ddneilson — CHANGES_REQUESTED** [review]

File: job_bundles/turntable_with_maya_arnold/template.yaml (hunk 1 above)
Comment (mwiebe) [reply job_bundles/turntable_with_maya_arnold/template.yaml]: Fixed

File: job_bundles/README.md (hunk 2 above)
Comment (mwiebe) [reply job_bundles/README.md]: Fixed

**Review by ddneilson — APPROVED** [review]

**Review by crowecawcaw — APPROVED** [review]

**Pull request merged**